### DISCUSSION TITLE
Dependabot spoofing to steal user's passwords

### DISCUSSION THREAD

**Discussion by moppman** [zeapo/Android-Password-Store#2701] [General]
I've just seen [this HN post](https://news.ycombinator.com/item?id=37680443), which is about malicious actors spoofing commits to  make them look like they're from Github's dependabot, and thus getting malicious code into projects.
Because this project makes heavy use of renovate (a similar bot from my understanding) and given the project's extra-sensitive nature, I thought it was worth mentioning here.

**Comment by msfjarvis** [comment]
Thanks for the heads-up. This attack vector shouldn't affect us since all legitimate Renovate PRs to APS originate from the repository itself, so it's very easy to spot malicious ones.
